extensions/markdown/README.md

```diff
@@ -0,0 +1,30 @@
+## Using Markdown-it Plugins
+
+To use a `markdown-it` plugin, goto the directory containing the `markdown-it` extension.  This will
+be found in the `extensions` folder in your vscode installation.  Install the `markdown-it` plugin
+as a node module.  For example, to install the `KaTeX` markdown-it extension you would simply type:
+
+``` sh
+npm install markdown-it-katex
+```
+
+Next, add the plugin into your [settings configuration](https://code.visualstudio.com/docs/customization/userandworkspace)
+under the `"markdown.plugs"` namespace.  This setting should contain a list for each `markdown-it` extension you wish
+to install.  The list should contain an object describing the name of the module, a list of styles that your extentsion
+may require, and options that you wish to pass to your extension.  For example, while installing the `markdown-it-katex`
+extension we would include this in your settings configuration:
+
+``` json
+    "markdown.plugins": [
+        {
+            "name": "markdown-it-katex",
+            "styles": [
```

Comment:
So the sequence will be: default style, user style, default plugin, custom
plugin. Correct ne if I made any mistake. If not, you may want to make this
clear for users :)

cbreeden <notifications@github.com>于2016年6月22日周三 08:12写道：

> In extensions/markdown/README.md
> <https://github.com/Microsoft/vscode/pull/7794#discussion_r67975515>:
>
> > +
> > +``` sh
> > +npm install markdown-it-katex
> > +```
> > +
> > +Next, add the plugin into your [settings configuration](https://code.visualstudio.com/docs/customization/userandworkspace)
> > +under the `"markdown.plugs"` namespace.  This setting should contain a list for each `markdown-it` extension you wish
> > +to install.  The list should contain an object describing the name of the module, a list of styles that your extentsion
> > +may require, and options that you wish to pass to your extension.  For example, while installing the `markdown-it-katex`
> > +extension we would include this in your settings configuration:
> > +
> > +``` json
> > +    "markdown.plugins": [
> > +        {
> > +            "name": "markdown-it-katex",
> > +            "styles": [
>
> Good point. with the current commit the order of the styles injects will
> be the same order as they are listed in the "markdown.styles"
> configuration, which occur after the default markdown styles from vscode.
> The order of the markdown plugins are also loaded in the same order as they
> are listed in the "markdown.plugins" configuration, which also occur after
> the default markdown plugins. Do you think this is sufficient flexibility?
> Should I add a remark in the README.md about this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/vscode/pull/7794/files/b7fdb0cf7e85d834251a97c843bf756b56ef8c51#r67975515>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA1heCPQ_nTktCVgCH4jrL8tbJsLrsPcks5qOH35gaJpZM4I32wm>
> .
>